cmd/minishift/cmd/config/config.go

```diff
@@ -116,6 +116,61 @@ var settings []Setting = []Setting{
 		name: config.ReminderWaitPeriodInHours,
```

Comment:
What's about we get rid of  _WantUpdateNotification_ and _ReminderWaitPeriodInHours_. AFAIC they are obsolete. WDYT?